openapi-specs/schema.json

```diff
@@ -4813,6 +4841,24 @@
             "schema": {
               "$ref": "#/definitions/BackupCreateRequest"
             }
+          },
+          {
+            "name": "X-Aws-Access-Key",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "X-Aws-Secret-Key",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "X-Aws-Session-Token",
+            "in": "header",
+            "required": false,
+            "type": "string"
```

Comment:
I think that we shouldn't add header definitions here in schema, there's lots of other headers that we use beside the AWS ones, so I would not include this information here

Reply:
can we remove this part?

Reply:
I'm not sure, the definitions file looks like the best place to put parameter definitions.  Other programs look in this file to generate calls, is there an actual problem with listing call definitions in the schema file?  Otherwise people need to manually intervene to make correct calls.

The main reason for using swagger is to have all the interface declarations in one place, why is this a bad thing?

Reply:
My point is that you have only added a very small subset of headers that Weaviate is supporting and we have never been adding them here to the swagger file, it's just that we don't do it, bc everytime we would introduce a new header we would have to regenerate the swagger and that is something that we don't do. If you want to add header definitions to schema.json, can we maybe do this is in a separate PR? and remove this change from this PR?

Reply:
The fact that we haven't done it in the past isn't really an engineering based reason for not doing it now.  And every time we introduce a new parameter we have to add it here, are headers really any different?  It's not clear to me why parameters in urls go here but headers don't. 

It's appropriate for this PR, because it supports this PR

Reply:
We haven't added module specific headers here bc those headers where only specific to a module and users could easily remove module without touching Weaviate's API definitions.

If you want to add support for Weaviate headers maybe you could do this in a follow up PR bc we support more then only those 3 headers (look [here](https://github.com/weaviate/weaviate/blob/main/usecases/config/config_handler.go#L313))

Reply:
sure, I'll move them to a follow up PR

Reply:
done